configs/test/gae/prod/app.yaml

```diff
@@ -56,3 +56,6 @@ basic_scaling:
 
 includes:
   - skip_files.yaml
+
+vpc_access_connector:
+  name: "projects/test-project/locations/us-central1/connectors/connector"
```

Comment:
We need to use "gce-zone" instead of us-central1 (just like project.yaml). That way it gets auto-subsituted to the right zone.

Reply:
Added "gae-region" and replaced with that. This needs to be in the gae region.